crypto/evp/evp_local.h

```diff
@@ -51,24 +52,27 @@ struct evp_cipher_ctx_st {
     int block_mask;
     unsigned char final[EVP_MAX_BLOCK_LENGTH]; /* possible final block */
 
-    /* Provider ctx */
-    void *provctx;
+    /*
+     * Opaque ctx returned from a providers cipher algorithm implementation
+     * OSSL_FUNC_cipher_newctx()
+     */
```

Comment:
This comment seems a bit bloated at this point.  How about `/* Algorithm-specific ctx */`?